install_config/master_node_configuration.adoc

```diff
@@ -1082,13 +1082,10 @@ If -1, there is no limit on requests.
 |A boolean to enable or disable dynamic provisioning. Default is *true*.
 
 |FSGroup
-|Can be specified to enable a quota on local storage use per unique FSGroup ID.
+|Can be specified to xref:node-config-volume-config[enable a quota on local storage] use per unique FSGroup ID.
```

Comment:
This is confusing. Do you mean "Can be specified to xref:node-config-volume-config[enable a quota on local storage]. You must enable quota for each unique FSGroup ID." ?

Reply:
The feature sets a blanket quota for all FSgroups on each node. 
"Enables xref:node-config-volume-config[local storage quotas] on each node for each FSGroup."